src/EnergyPlus/MixedAir.cc

```diff
@@ -4232,14 +4265,72 @@ namespace MixedAir {
 		if ( OutAirSignal > OutAirMinFrac && OutAirSignal < 1.0 && this->MixMassFlow > VerySmallMassFlow && this->ControllerType_Num == ControllerOutsideAir && ! AirLoopNightVent ) {
-			Par( 1 ) = this->MixNode;
-			Par( 2 ) = this->RetNode;
-			Par( 3 ) = this->InletNode;
-			Par( 4 ) = this->MixMassFlow;
-			SolveRegulaFalsi( Acc, MaxIte, SolFla, OASignal, MixedAirControlTempResidual, OutAirMinFrac, 1.0, Par );
-			if ( SolFla < 0 ) {
-				OASignal = OutAirSignal;
+
+			if ( AirLoopNum > 0 ) {
+
+				if( OutsideAirSys( AirLoopControlInfo( AirLoopNum ).OASysNum ).NumComponents == 1 ) {
+					// no need to simulate OA System if only a mixer is used in the OutsideAirSystem
+
+					Par( 1 ) = this->MixNode;
+					Par( 2 ) = this->RetNode;
+					Par( 3 ) = this->InletNode;
+					Par( 4 ) = this->MixMassFlow;
+					SolveRegulaFalsi( Acc, MaxIte, SolFla, OASignal, MixedAirControlTempResidual, OutAirMinFrac, 1.0, Par );
+					if( SolFla < 0 ) {
+						OASignal = OutAirSignal;
+					}
+
+				} else {
+
+					// simulate OA System if equipment exists other than the mixer (e.g., heating/cooling coil, HX, ect.)
+
+					// 1 - check min OA flow result
+					Node( this->OANode ).MassFlowRate = max( this->ExhMassFlow, OutAirMinFrac * Node( this->MixNode ).MassFlowRate );
+					Node( this->RelNode ).MassFlowRate = max( Node( this->OANode ).MassFlowRate - this->ExhMassFlow, 0.0 );
+					// save actual OA flow frac for use as min value for RegulaFalsi call
+					minOAFrac = max( OutAirMinFrac, Node( this->OANode ).MassFlowRate / this->MixMassFlow );
+					SimOASysComponents( AirLoopControlInfo( AirLoopNum ).OASysNum, FirstHVACIteration, AirLoopNum );
+					lowFlowResiduum = Node( this->MixNode ).TempSetPoint - Node( this->MixNode ).Temp;
+
+					// 2 - check max OA flow result
+					Node( this->OANode ).MassFlowRate = max( this->ExhMassFlow, Node( this->MixNode ).MassFlowRate );
+					Node( this->RelNode ).MassFlowRate = max( Node( this->OANode ).MassFlowRate - this->ExhMassFlow, 0.0 );
+					SimOASysComponents( AirLoopControlInfo( AirLoopNum ).OASysNum, FirstHVACIteration, AirLoopNum );
+					highFlowResiduum = Node( this->MixNode ).TempSetPoint - Node( this->MixNode ).Temp;
+
+					// 3 - test to ensure RegulaFalsi can find an answer
+					if( ( sign( lowFlowResiduum ) == sign( highFlowResiduum ) ) ) {
+						OASignal = OutAirSignal;
+					} else {
+						// 4 - find result
+						Par( 1 ) = this->MixNode;
+						Par( 2 ) = this->RelNode;
+						Par( 3 ) = this->OANode;
+						Par( 4 ) = this->MixMassFlow;
+						Par( 5 ) = 0.0;
+						if( FirstHVACIteration ) Par( 5 ) = 1.0;
+						Par( 6 ) = double( AirLoopNum );
+
+						SolveRegulaFalsi( ( Acc / 10.0 ), MaxIte, SolFla, OASignal, MultiCompControlTempResidual, minOAFrac, 1.0, Par );
+						if( SolFla < 0 ) { // if RegulaFalsi fails to find a solution, returns -1 or -2, set to existing OutAirSignal
+							OASignal = OutAirSignal;
+						}
+					}
+				}
+
+			} else {
```

Comment:
Seems like this block is repeated from the first section of this else block, so redundant, but maybe it is intentional for code clarity?